content/cli/cloud/cluster.md

```diff
@@ -4,4 +4,107 @@ menu: Cluster
 weight: 30
 ---
 
-TBD
\ No newline at end of file
+The `fluvio cloud cluster` family of commands is used to create, delete, and troubleshoot Fluvio clusters in cloud.
+
+%copy first-line%
+```bash
+$  fluvio cloud cluster -h
+```
```

Comment:
No need to separate these 2 blocks (unless you want gray formatting).

Reply:
I just copied that format from `fluvio cloud connectors` page. Updated now